regen-network/regen-web
# feat: APP-19 generate project slug based on project name

## Description

https://regennetwork.atlassian.net/browse/APP-19?focusedCommentId=10108



---

### Author Checklist

_All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues._

I have...


### How to test



1. From https://deploy-preview-2335--regen-marketplace.netlify.app/ login and create a new project
2. Go through the creation process. After publishing you should see that the project slug has been generated based on the project name
3. Create another project with the same name, the generated slug should be the same but suffixed with "-2" to avoid any duplicate slugs
4. Edit a published project name, it should not update the project slug

### Reviewers Checklist

_All items are required. Please add a note if the item is not applicable and please **add
your handle next to the items reviewed if you only reviewed selected items**._

I have...
